src/main/resources/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder/config.jelly

```diff
@@ -26,6 +26,9 @@
     <f:entry field="clusterName" title="${%Cluster}">
         <f:select/>
     </f:entry>
+    <f:entry field="namespace" title="${%Namespace}">
+        <f:textbox default="default"/>
```

Comment:
This should be empty by default as to match expectations when doing a normal `kubectl apply -f blah/`

Reply:
Done.